modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentNodeHelper.java

```diff
@@ -78,6 +80,11 @@ public static ContentIds toContentIds( final NodeIds nodeIds )
         return ContentIds.from( nodeIds.stream().map( nodeId -> ContentId.from( nodeId.toString() ) ).collect( Collectors.toList() ) );
     }
 
+    private static String getRoot()
+    {
+        final NodePath nodePath = (NodePath) ContextAccessor.current().getAttribute(ContentConstants.CONTENT_ROOT_PATH_ATTRIBUTE);
```

Comment:
I don't see where it is set. And keep a secret "String" for now. something that is hard to collide with `__content_root_path__`